core/src/main/java/uk/gov/justice/services/core/audit/LocalAuditInterceptor.java

```diff
@@ -19,16 +19,18 @@
 
     @Override
     public InterceptorContext process(final InterceptorContext interceptorContext, final InterceptorChain interceptorChain) {
-        recordAudit(interceptorContext.inputEnvelope());
+        final String component = (String) interceptorContext.getInputParameter("component")
```

Comment:
could we add getComponent() on InterceptorContext?

Presumably it is used elsewhere already so makes sense to have it in one place and also not expose the "component" value?